src/qt/sendcoinsdialog.cpp

```diff
@@ -186,6 +187,11 @@ void SendCoinsDialog::setModel(WalletModel *_model)
         // set default rbf checkbox state
         ui->optInRBF->setCheckState(Qt::Checked);
 
+        if (model->privateKeysDisabled()) {
+            ui->sendButton->setText(tr("C&reate Unsigned"));
```

Comment:
Perhaps use `"Cr&eate Unsigned"` so the "E" shortcut matches with `"S&end"`

Reply:
Done